src/correctionlib/util.py

```diff
@@ -1,15 +1,19 @@
+import importlib.resources
 import pathlib
 import sys
 
+if sys.version_info >= (3, 11):
+    path_return_type = importlib.resources.abc.Traversable
+else:
+    path_return_type = pathlib.Path
 
-def this_module_path() -> pathlib.Path:
+def this_module_path() -> path_return_type
```

Comment:
cast to `Path` in the return?

Reply:
Traversable is not necessary an actual path.

Reply:
What is very unclear is which version Traversable becomes the return type in. Augh.